src/main/resources/sample.elasticapm.properties

```diff
@@ -0,0 +1,7 @@
+service_name=metis-sandbox-rest-dev
```

Comment:
Also can we add this configuration in the spring configuration file, or is that not possible?
I'm saying this because although splitting the configuration can be better organizing it but when configuring stuff we have to handle two separate files and many times it becomes more annoying than actually helping.